### PR TITLE
Tests for CldOgImage

### LINKED ISSUE #145: add tests for CldOgImage
We use [vitest](https://vitest.dev/) for our testing suite, and the tests are located in [packages/svelte-cloudinary/tests](./packages/svelte-cloudinary/tests). We just need some simple tests that cover basic functionality for now, similar to the example below:

Example: https://github.com/cloudinary-community/svelte-cloudinary/blob/main/packages/svelte-cloudinary/tests/CldImage/CldImage.test.ts

### PR DESCRIPTION
**Pull request by ovindu-a**
# Description

I have added a few test to make sure the meta tags for the Open graph Image are rendered

## Issue Ticket Number

Fixes #145 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Fix or improve the documentation
- [ ] This change requires a documentation update

# Checklist

- [x] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have created an [issue](https://github.com/cloudinary-community/svelte-cloudinary/issues) ticket for this PR
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/cloudinary-community/svelte-cloudinary/pulls) for the same update/change?
- [x] I have performed a self-review of my own code
- [x] I have run tests locally to ensure they all pass
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes needed to the documentation

### CODE REVIEW

File: packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        });
+    
+        const metaTag = document.querySelector('meta[property="og:image"]');
+        
+        if (metaTag instanceof HTMLMetaElement) {
+          expect(metaTag.content).toContain('https://res.cloudinary.com');
+          expect(metaTag.content).toContain('og_image');
+        } else {
+          throw new Error('og:image meta tag not found');
+        }
```
Comment (ghostdevv) [inline packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts:26]: Could you change all these tests to type the metaTag, and use the expect instance of checker - something like this:

```js
const metaTag = document.querySelector<HTMLMetaElement>('meta[property="og:image"]');

expect(metaTag).toBeInstanceOf(HTMLMetaElement)
expect(metaTag?.content).toContain('https://res.cloudinary.com');
expect(metaTag?.content).toContain('og_image');
```

File: packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	afterEach(() => {
+		cleanup();
+	});
+
+    it('should generate the correct URL for Cloudinary Open Graph image in the og:image meta tag', () => {
+        render(CldOgImage, {
+          props: {
+            src: 'og_image',
+
+            config: { cloud: { cloudName: 'testing-og' } },
```
Comment (ghostdevv) [inline packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts:15]: could you fix the type errors on these, I believe it's just missing an alt prop which you can like `alt: ''`

File: packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts (hunk 1 above)
Comment (ghostdevv) [inline packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts:26]: Could you change all these tests to type the metaTag, and use the expect instance of checker - something like this:

```js
const metaTag = document.querySelector<HTMLMetaElement>('meta[property="og:image"]');

expect(metaTag).toBeInstanceOf(HTMLMetaElement)
expect(metaTag?.content).toContain('https://res.cloudinary.com');
expect(metaTag?.content).toContain('og_image');
```

**Review by ghostdevv — CHANGES_REQUESTED** [review]

**Comment by ghostdevv** [comment]
> If there is no changes I will add the changeset

we don't need a changeset for this one since it's just tests, and won't be published on npm

File: packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts (hunk 1 above)
Comment (ovindu-a) [reply packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts:26]: Sure I will make the changes

**Comment by ovindu-a** [comment]
@ghostdevv I have made the requested changes in the above commit

File: packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+	it('should work with global config from environment variables', () => {
+		const cloudName = crypto.randomUUID();
+		vi.stubEnv('VITE_CLOUDINARY_CLOUD_NAME', cloudName);
+
+		render(CldOgImage, {
+			props: {
+				src: 'og_image',
+				alt: ''
+			},
```
Comment (ghostdevv) [inline packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts:55]:
```suggestion
				alt: '',
			},
```

**Review by ghostdevv — CHANGES_REQUESTED** [review]
oops, lint error

**Review by ghostdevv — APPROVED** [review]
Ok, perfect!! Thank you

**Pull request merged**